packages/suite/src/components/wallet/WalletLayout/components/CoinjoinAccountDiscovery/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+
+import { toggleRememberDevice } from '@suite-actions/suiteActions';
+import { useDevice } from '@suite-hooks';
+import { isDeviceRemembered } from '@suite-utils/device';
+import { CoinjoinAccountDiscoveryProgress } from './CoinjoinAccountDiscoveryProgress';
+import { RememberWallet } from './RememberWallet';
+
+export const CoinjoinAccountDiscovery = () => {
+    const [shouldRememberCardPersist, setShouldRememberCardPersist] = useState(false);
+
+    const dispatch = useDispatch();
+
+    const { device } = useDevice();
+
+    const deviceRemembered = isDeviceRemembered(device);
```

Comment:
device is possibly undefined here, type error

Reply:
Sorry, I just notice and force-pushed a fix.